src/main/kotlin/fr/social/gouv/agora/infrastructure/qag/repository/QagInfoMapper.kt

```diff
@@ -31,4 +33,25 @@ class QagInfoMapper {
         )
     }
 
+    fun toDto(domain: QagInserting): QagDTO? {
+        return try {
+            QagDTO(
+                id = UUID.randomUUID(),
```

Comment:
Le `id` dans le domain n'est même pas utilisé ici, c'est qu'il n'est pas nécessaire dans le domain ;)

Reply:
you're right!

Reply:
En vrai, on peut mettre le UUID 0

Reply:
done